web/src/components/common/LazyWrapper.tsx

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import {useInView} from 'react-hook-inview';
+
+const LazyWrapper: React.FC = ({children}) => {
+  const [ref, isInView] = useInView({
+    unobserveOnEnter: true,
+    rootMargin: '30px',
```

Comment:
What is this? Is it used to prefetch before children is visible in the window?
Can we add a function param for this with default value 30px?

Reply:
@ifhuang Yep that's right, the `child` will be fetched 30px before it is in view! Ok sure I can make it a prop :)